x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/annotations/Annotation.java

```diff
@@ -14,15 +14,44 @@
 import org.elasticsearch.common.xcontent.ObjectParser;
 import org.elasticsearch.common.xcontent.ToXContentObject;
 import org.elasticsearch.common.xcontent.XContentBuilder;
+import org.elasticsearch.common.xcontent.XContentParser;
 import org.elasticsearch.xpack.core.common.time.TimeUtils;
 import org.elasticsearch.xpack.core.ml.job.config.Job;
 
 import java.io.IOException;
 import java.util.Date;
+import java.util.Locale;
 import java.util.Objects;
 
 public class Annotation implements ToXContentObject, Writeable {
 
+    public enum Type {
+        ANNOTATION;
```

Comment:
I think you should also add `COMMENT` here, to keep the UI and backend in sync - the UI defines it here https://github.com/elastic/kibana/blob/995c2cb725a5588fc843b35895d48408036ed8ca/x-pack/plugins/ml/common/types/annotations.ts#L75